econml/_ortho_learner.py

```diff
@@ -552,127 +571,151 @@ def fit(self, Y, T, X=None, W=None, Z=None, *, sample_weight=None, sample_var=No
         inference: string, :class:`.Inference` instance, or None
             Method for performing inference.  This estimator supports 'bootstrap'
             (or an instance of :class:`.BootstrapInference`).
```

Comment:
I think at this point yes, it makes sense to me! Let's do it in a subsequent PR as it would need to add relevant deprecation code, which seems non-trivial.

Reply:
Also I suspect ti would need to be made a variable that is initialized by the BaseCateEstimator, and not any children and so it would need a lot of change. For now actulaly it seems an ok design, even for 0.9

Reply:
Sure.